STYLE.md

```diff
@@ -30,7 +30,8 @@
   fiasco](https://isocpp.org/wiki/faq/ctors#static-init-order-on-first-use) for
   how to best handle this.
 * API-level comments should follow normal Doxygen conventions. Use `@param` to describe
-  parameters, `@return <return-type>` for return values.
+  parameters, `@return <return-type>` for return values. Internal comments for
```

Comment:
I'll register my mild opposition to this here.  It seems if we are going through the work to document internal methods and member variables then we might as well follow the Doxygen style.  This is largely an academic point for member variables but for methods doxygen provides a structured way of documenting arguments and return semantics that is just as relevant for private methods.  The default Doxygen config can be set up to ignore private docs, but for someone new to the code base being able to run Doxygen with the call graph feature and all private documentation enabled seems a good way to create browsable overview documentation (I've used this in past projects, but admittedly we had more extensive documentation of private data and methods from the beginning).